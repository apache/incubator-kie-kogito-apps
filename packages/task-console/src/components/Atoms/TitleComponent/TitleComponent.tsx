import React from 'react';

const TitleComponent = () => {
  return <div> Task-Console</div>;
};

export default TitleComponent;
