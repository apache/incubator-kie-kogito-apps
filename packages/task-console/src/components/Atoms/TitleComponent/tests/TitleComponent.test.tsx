import React from 'react';
import { shallow } from 'enzyme';
import TitleComponent from '../TitleComponent';

describe('Title Component Test cases', () => {
  it('testing snapshot', () => {
    const wrapper = shallow(<TitleComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
