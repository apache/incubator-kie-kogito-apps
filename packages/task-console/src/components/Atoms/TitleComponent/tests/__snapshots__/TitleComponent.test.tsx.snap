// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component Test cases testing snapshot 1`] = `
<div>
  Task-Console
</div>
`;
